phys/module_mp_thompson.F

```diff
@@ -3268,8 +3267,10 @@ subroutine mp_thompson (qv1d, qc1d, qi1d, qr1d, qs1d, qg1d, ni1d, &
            t4_vts = Kap1*Mrat**mu_s*csg(7)*ils2**cse(7)
            vts = rhof(k)*av_s * (t1_vts+t2_vts)/(t3_vts+t4_vts)
            if (temp(k).gt. (T_0+0.1)) then
-            vtsk(k) = MAX(vts*vts_boost(k),                             &
-     &                vts*((vtrk(k)-vts*vts_boost(k))/(temp(k)-T_0)))
+!           vtsk(k) = MAX(vts*vts_boost(k),                             &
```

Comment:
If this is the old way, should we just remove the old code, instead of commenting it out?

Reply:
@davegill  OK, I did that and re-committed changes and pushed to your branch.